README.md

```diff
@@ -77,19 +78,43 @@ Parameters for the script are controlled by the following environment variables:
 
 It is recommended to execute the runner script through the `make run-tests` make target.
 
-Example:
+Example running the end-to-end test case:
 ```
 $ export KUBECONFIG=/path/to/kubeconfig
 $ export DUMP_FAILED_TESTS=true
 $ export REPORTS_DUMP_DIR=/tmp/nvidia-ci-logs-dir
 $ export TEST_FEATURES="nvidiagpu"
 $ export TEST_LABELS='nvidia-ci,gpu'
+$ export TEST_TRACE=true
 $ export VERBOSE_LEVEL=100
 $ export NVIDIAGPU_GPU_MACHINESET_INSTANCE_TYPE="g4dn.xlarge"
 $ export NVIDIAGPU_CATALOGSOURCE="certified-operators"
 $ export NVIDIAGPU_SUBSCRIPTION_CHANNEL="v23.9"
 $ make run-tests                    
 Executing eco-gotests test-runner script
 scripts/test-runner.sh
-ginkgo -timeout=24h --keep-going --require-suite -r --label-filter="nvidia-ci,gpu" ./tests/nvidiagpu
+ginkgo -timeout=24h --keep-going --require-suite -r -vv --trace --label-filter="nvidia-ci,gpu" ./tests/nvidiagpu
+```
+
+Example running the GPU operator upgrade testcase (from v23.6 to v24.3) after the end-end testcase.
+Note:  you must run the end-to-end testcase first to deploy a previous version, set NVIDIAGPU_CLEANUP=false, 
```

Comment:
Are the tests running sequentially? How do we enforce that when someone runs `--label-filter="nvidia-ci,gpu,operator-upgrade"`, then `nvidia-ci` and `gpu` run before `operator-upgrade`?

Reply:
@ybettan, yes the upgrade testcase will run sequencially after the testcase that deploys the gpu operator, it is the second testcase in the testcase go file.   In our CI we can set the labels needed in the job parameters, for the `operator-upgrade` and the channel to deploy, we also need the channel to upgrade to.